src/App.tsx

```diff
@@ -129,75 +203,95 @@ export const App: React.FC = () => {
               ×
             </button>
 
-            {/* 'is-active' class puts this modal on top of the todo */}
+            {/* 'is-active' class puts this modal on top of the todo
             <div data-cy="TodoLoader" className="modal overlay is-active">
               <div className="modal-background has-background-white-ter" />
               <div className="loader" />
             </div>
-          </div>
+          </div> */}
         </section>
 
         {/* Hide the footer if there are no todos */}
-        <footer className="todoapp__footer" data-cy="Footer">
-          <span className="todo-count" data-cy="TodosCounter">
-            3 items left
-          </span>
-
-          {/* Active link should have the 'selected' class */}
-          <nav className="filter" data-cy="Filter">
-            <a
-              href="#/"
-              className="filter__link selected"
-              data-cy="FilterLinkAll"
-            >
-              All
-            </a>
+        {todos.length !== 0 && (
```

Comment:
```suggestion
        {!!todos.length && (
```
You use it twice so you can save it to a variable